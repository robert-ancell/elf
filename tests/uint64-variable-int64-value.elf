int64 v = 9223372036854775807
uint64 value = v
print (value)
