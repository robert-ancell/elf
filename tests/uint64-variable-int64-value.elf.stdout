Line 1:
int64 v = 18446744073709551615
      ^
Variable is of type int64, but value is of type uint64
